pkg/providers/gateway/gateway.go

```diff
@@ -206,6 +207,49 @@ func (c *gatewayController) onAdd(obj interface{}) {
 }
 
 func (c *gatewayController) onUpdate(oldObj, newObj interface{}) {
+	oldGateway, ok := oldObj.(*gatewayv1beta1.Gateway)
+	if !ok {
+		log.Errorw("failed to convert old object to gateway",
+			zap.Any("obj", oldObj),
+		)
+		return
+	}
+
+	newGateway, ok := newObj.(*gatewayv1beta1.Gateway)
+	if !ok {
+		log.Errorw("failed to convert new object to gateway",
+			zap.Any("new obj", newObj),
+		)
+		return
+	}
+
+	if reflect.DeepEqual(oldGateway.Spec, newGateway.Spec) {
```

Comment:
```suggestion
	if oldHTTPRoute.ResourceVersion >= newHTTPRoute.ResourceVersion {
```
I just wanted to know if we can change it to this one or previous one is okay.